src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210424205311_AddDbTasksTable.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace LT.DigitalOffice.ProjectService.Data.Provider.MsSql.Ef.Migrations
+{
+    [DbContext(typeof(ProjectServiceDbContext))]
+    [Migration("20210424205311_AddDbTasksTable")]
+    public class _20210424205311_AddDbTasksTable : Migration
```

Comment:
Please, rename class to AddDbTasksTable

Reply:
Spartak said it is ok name it with numbers according to file name, but ok, I changed it